dcp_prototype/backend/browser/requirements-dev.txt

```diff
@@ -0,0 +1 @@
+chalice==1.12.0
```

Comment:
can we move this to the root requirements-dev.txt? I don't think we need requirements-dev.txt within each directory.

Reply:
I think this is ok. Everything at the top level should be as vague and unassuming about the underlying services as possible. I don't want these top level library requirements to get in the way of components developing their services, especially libraries that are version locked. Linters, and code sanitation tool are the only dependencies that should live at the to level requirements-dev.txt. I plan to use multiple python environments depending on the code i'm working on. I thinking keeping requirements as local as possible will help facilitate this.